### ISSUE TITLE
Microsoft Edge UIA Framework ID changed

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/nvda#5955]
As of Windows 10 build 14328, the UIA framework ID for Edge is now MicrosoftEdge, rather than InternetExplorer. 
NVDA must be changed to detect this so as to provide browse mode for later builds of Edge.
Issue #5901 also depends on this.

**Comment by nvaccessAuto** [comment]
Incubated in d5f37327edb7804203dcc0aa977d4cde7d09a08c.

**Comment by josephsl** [comment]
Hi,

I'm not sure if the below issue has anything to do with this...

A variant of #5901: In build 14361, if you restart NVDA, NVDA will fail to notice UIA item selection event.

STR:

1. Tell NVDA to start after you log onto your computer.
2. Restart your computer.
3. Open Start, then search for something.
4. Restart NvDA and repeat step 3.

Expected: NVDA will notice UIA item selection event.
Actual: No search results announcement.

Setup:

* NVDA version: next.13351
* OS: Windows 10 Build 14361
* Reproducible: Yes.

Other than this, UIA is functioning as expected - recognizes various UIA controls, for instance. Thanks.

**Comment by jcsteh** [comment]
I don't think this is related. However, it's worth noting that this bug 
is really weird and suggests some serious UIA brokenness (surprise 
surprise). If you start Narrator while looking at search results, the 
events magically start firing again.

**Comment by josephsl** [comment]
Hi,
Last issue fixed as of build 14366. Thanks.
